.ci/lib.sh

```diff
@@ -324,30 +324,37 @@ cleanup_network_interface() {
 }
 
 gen_clean_arch() {
-	# Set up some vars
-	stale_process_union=( "docker-containerd-shim" )
-	#docker supports different storage driver, such like overlay2, aufs, etc.
-	docker_storage_driver=$(timeout ${KATA_DOCKER_TIMEOUT} docker info --format='{{.Driver}}')
-	stale_docker_mount_point_union=( "/var/lib/docker/containers" "/var/lib/docker/${docker_storage_driver}" )
-	stale_docker_dir_union=( "/var/lib/docker" )
-	stale_kata_dir_union=( "/var/lib/vc" "/run/vc" "/usr/share/kata-containers" "/usr/share/defaults/kata-containers" )
-
-	info "kill stale process"
-	kill_stale_process
-	if [ -z "${USE_PODMAN}" ]; then
-		info "delete stale docker resource under ${stale_docker_dir_union[@]}"
-		delete_stale_docker_resource
+	if [[ $CI_JOB != "METRICS" ]]; then
```

Comment:
Could you add a comment explaining why METRICS is different here? Maybe something like

```
# The metrics CI on bare metal generates errors during cleanup because it does not install packages
```